x-pack/plugin/eql/qa/common/src/main/resources/data/extra.data

```diff
@@ -2,26 +2,31 @@
  {
   "@timestamp": "1",
   "event_type": "REQUEST",
-  "transID": 1234,
-  "sequence": 1
+  "transID": [1234, 599999],
```

Comment:
Please add some entries where the multi keys:
1. are in random order 
2. have duplicates.
3. contain null entry

Reply:
Yes, more tests are needed here (also unit tests), I just wanted a feedback first, to make sure that there is an agreement on the expected outcome and on the general approach.